App Crashing on Tablets

**Summary:** 
App is crashing in Tablet, working fine on Phones

**Steps to reproduce:** 
Launch the app in a Tablet

**System logs:**
```
2019-03-27 22:33:45.813 2231-2231/? I/Zygote: seccomp disabled by setenforce 0
2019-03-27 22:33:45.817 2231-2231/? I/rw.commons.bet: Late-enabling -Xcheck:jni
2019-03-27 22:33:45.915 2231-2231/? W/rw.commons.bet: Unexpected CPU variant for X86 using defaults: x86
2019-03-27 22:33:46.261 2231-2231/fr.free.nrw.commons.beta I/rw.commons.bet: The ClassLoaderContext is a special shared library.
2019-03-27 22:33:47.109 2231-2231/fr.free.nrw.commons.beta E/LoadedApk: Unable to instantiate appComponentFactory
    java.lang.ClassNotFoundException: Didn't find class "fr.free.nrw.commons.beta.commons" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/base.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_dependencies_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_resources_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_0_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_1_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_2_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_3_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_4_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_5_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_6_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_7_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_8_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/base.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_dependencies_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_resources_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_0_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_1_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_2_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_3_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_4_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_5_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_6_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_7_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_8_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_9_apk.apk!/lib/x86, /system/lib, /system/vendor/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.app.LoadedApk.createAppFactory(LoadedApk.java:226)
        at android.app.LoadedApk.createOrUpdateClassLoaderLocked(LoadedApk.java:731)
        at android.app.LoadedApk.getClassLoader(LoadedApk.java:810)
        at android.app.LoadedApk.getResources(LoadedApk.java:1032)
        at android.app.ContextImpl.createAppContext(ContextImpl.java:2345)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5760)
        at android.app.ActivityThread.access$1100(ActivityThread.java:200)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1651)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6680)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
    	Suppressed: java.io.IOException: No original dex files found for dex location /data/app/fr.free.nrw.comm
2019-03-27 22:33:47.119 2231-2231/fr.free.nrw.commons.beta W/rw.commons.bet: JIT profile information will not be recorded: profile file does not exits.
2019-03-27 22:33:47.123 2231-2231/fr.free.nrw.commons.beta I/chatty: uid=10071(fr.free.nrw.commons.beta) identical 10 lines
2019-03-27 22:33:47.123 2231-2231/fr.free.nrw.commons.beta W/rw.commons.bet: JIT profile information will not be recorded: profile file does not exits.
2019-03-27 22:33:47.705 2231-2231/fr.free.nrw.commons.beta I/InstantRun: starting instant run server: is main process
2019-03-27 22:33:48.433 2231-2242/fr.free.nrw.commons.beta I/rw.commons.bet: Background concurrent copying GC freed 12378(2MB) AllocSpace objects, 3(52KB) LOS objects, 49% free, 1983KB/3MB, paused 850us total 540.346ms
2019-03-27 22:33:48.599 2231-2231/fr.free.nrw.commons.beta I/ACRA: ACRA is enabled for fr.free.nrw.commons.beta, initializing...
2019-03-27 22:33:48.932 2231-2231/fr.free.nrw.commons.beta D/NetworkSecurityConfig: No Network Security Config specified, using platform default
2019-03-27 22:33:49.257 2231-2231/fr.free.nrw.commons.beta D/SoLoader: init start
2019-03-27 22:33:49.257 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding system library source: /vendor/lib
2019-03-27 22:33:49.258 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding system library source: /system/lib
2019-03-27 22:33:49.258 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding application source: com.facebook.soloader.DirectorySoSource[root = /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/lib/x86 flags = 0]
2019-03-27 22:33:49.264 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source from : com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-main flags = 1]
2019-03-27 22:33:49.264 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup sources from split apks
2019-03-27 22:33:49.265 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-0 flags = 1]
2019-03-27 22:33:49.265 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-1 flags = 1]
2019-03-27 22:33:49.265 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-2 flags = 1]
2019-03-27 22:33:49.265 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-3 flags = 1]
2019-03-27 22:33:49.265 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-4 flags = 1]
2019-03-27 22:33:49.265 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-5 flags = 1]
2019-03-27 22:33:49.265 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-6 flags = 1]
2019-03-27 22:33:49.265 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-7 flags = 1]
2019-03-27 22:33:49.266 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-8 flags = 1]
2019-03-27 22:33:49.266 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-9 flags = 1]
2019-03-27 22:33:49.266 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-10 flags = 1]
2019-03-27 22:33:49.266 2231-2231/fr.free.nrw.commons.beta D/SoLoader: adding backup source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-11 flags = 1]
2019-03-27 22:33:49.266 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.DirectorySoSource[root = /system/lib flags = 2]
2019-03-27 22:33:49.266 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.DirectorySoSource[root = /system/vendor/lib flags = 2]
2019-03-27 22:33:49.266 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.DirectorySoSource[root = /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/lib/x86 flags = 0]
2019-03-27 22:33:49.266 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-11 flags = 1]
2019-03-27 22:33:49.267 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-11
2019-03-27 22:33:49.278 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-11
2019-03-27 22:33:49.279 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-11
2019-03-27 22:33:49.279 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-10 flags = 1]
2019-03-27 22:33:49.279 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-10
2019-03-27 22:33:49.281 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-10
2019-03-27 22:33:49.281 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-10
2019-03-27 22:33:49.282 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-9 flags = 1]
2019-03-27 22:33:49.282 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-9
2019-03-27 22:33:49.283 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-9
2019-03-27 22:33:49.283 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-9
2019-03-27 22:33:49.283 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-8 flags = 1]
2019-03-27 22:33:49.283 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-8
2019-03-27 22:33:49.286 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-8
2019-03-27 22:33:49.286 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-8
2019-03-27 22:33:49.287 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-7 flags = 1]
2019-03-27 22:33:49.287 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-7
2019-03-27 22:33:49.289 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-7
2019-03-27 22:33:49.289 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-7
2019-03-27 22:33:49.289 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-6 flags = 1]
2019-03-27 22:33:49.290 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-6
2019-03-27 22:33:49.291 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-6
2019-03-27 22:33:49.291 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-6
2019-03-27 22:33:49.291 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-5 flags = 1]
2019-03-27 22:33:49.293 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-5
2019-03-27 22:33:49.294 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-5
2019-03-27 22:33:49.294 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-5
2019-03-27 22:33:49.294 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-4 flags = 1]
2019-03-27 22:33:49.294 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-4
2019-03-27 22:33:49.295 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-4
2019-03-27 22:33:49.295 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-4
2019-03-27 22:33:49.296 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-3 flags = 1]
2019-03-27 22:33:49.296 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-3
2019-03-27 22:33:49.297 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-3
2019-03-27 22:33:49.297 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-3
2019-03-27 22:33:49.297 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-2 flags = 1]
2019-03-27 22:33:49.297 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-2
2019-03-27 22:33:49.299 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-2
2019-03-27 22:33:49.299 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-2
2019-03-27 22:33:49.299 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-1 flags = 1]
2019-03-27 22:33:49.299 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-1
2019-03-27 22:33:49.312 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-1
2019-03-27 22:33:49.312 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-1
2019-03-27 22:33:49.313 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-0 flags = 1]
2019-03-27 22:33:49.313 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-0
2019-03-27 22:33:49.314 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-0
2019-03-27 22:33:49.314 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-0
2019-03-27 22:33:49.315 2231-2231/fr.free.nrw.commons.beta D/SoLoader: Preparing SO source: com.facebook.soloader.ApkSoSource[root = /data/data/fr.free.nrw.commons.beta/lib-main flags = 1]
2019-03-27 22:33:49.315 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: locked dso store /data/user/0/fr.free.nrw.commons.beta/lib-main
2019-03-27 22:33:49.329 2231-2231/fr.free.nrw.commons.beta I/fb-UnpackingSoSource: dso store is up-to-date: /data/user/0/fr.free.nrw.commons.beta/lib-main
2019-03-27 22:33:49.329 2231-2231/fr.free.nrw.commons.beta V/fb-UnpackingSoSource: releasing dso store lock for /data/user/0/fr.free.nrw.commons.beta/lib-main
2019-03-27 22:33:49.331 2231-2231/fr.free.nrw.commons.beta D/SoLoader: init finish: 16 SO sources prepared
2019-03-27 22:33:49.331 2231-2231/fr.free.nrw.commons.beta D/SoLoader: init exiting
2019-03-27 22:33:49.496 2231-2251/fr.free.nrw.commons.beta I/stetho: Listening on @stetho_fr.free.nrw.commons.beta_devtools_remote
2019-03-27 22:33:49.620 2231-2254/fr.free.nrw.commons.beta D/libEGL: Emulator has host GPU support, qemu.gles is set to 1.
2019-03-27 22:33:49.617 2231-2231/fr.free.nrw.commons.beta I/fr.free.nrw.commons.beta: type=1400 audit(0.0:735): avc: denied { write } for comm=45474C20496E6974 name="property_service" dev="tmpfs" ino=8350 scontext=u:r:untrusted_app:s0:c71,c256,c512,c768 tcontext=u:object_r:property_socket:s0 tclass=sock_file permissive=1
2019-03-27 22:33:49.629 2231-2254/fr.free.nrw.commons.beta D/vndksupport: Loading /vendor/lib/egl/libGLES_emulation.so from current namespace instead of sphal namespace.
2019-03-27 22:33:49.625 2231-2231/fr.free.nrw.commons.beta I/fr.free.nrw.commons.beta: type=1400 audit(0.0:736): avc: denied { connectto } for comm=45474C20496E6974 path="/dev/socket/property_service" scontext=u:r:untrusted_app:s0:c71,c256,c512,c768 tcontext=u:r:init:s0 tclass=unix_stream_socket permissive=1
2019-03-27 22:33:49.640 2231-2254/fr.free.nrw.commons.beta E/libEGL: load_driver(/vendor/lib/egl/libGLES_emulation.so): dlopen failed: library "/vendor/lib/egl/libGLES_emulation.so" not found
2019-03-27 22:33:49.640 2231-2254/fr.free.nrw.commons.beta D/vndksupport: Loading /vendor/lib/egl/libEGL_emulation.so from current namespace instead of sphal namespace.
2019-03-27 22:33:49.709 2231-2254/fr.free.nrw.commons.beta D/libEGL: loaded /vendor/lib/egl/libEGL_emulation.so
2019-03-27 22:33:49.729 2231-2254/fr.free.nrw.commons.beta D/vndksupport: Loading /vendor/lib/egl/libGLESv1_CM_emulation.so from current namespace instead of sphal namespace.
2019-03-27 22:33:49.746 2231-2254/fr.free.nrw.commons.beta D/libEGL: loaded /vendor/lib/egl/libGLESv1_CM_emulation.so
2019-03-27 22:33:49.804 2231-2254/fr.free.nrw.commons.beta D/vndksupport: Loading /vendor/lib/egl/libGLESv2_emulation.so from current namespace instead of sphal namespace.
2019-03-27 22:33:49.821 2231-2254/fr.free.nrw.commons.beta D/libEGL: loaded /vendor/lib/egl/libGLESv2_emulation.so
2019-03-27 22:33:49.883 2231-2242/fr.free.nrw.commons.beta I/rw.commons.bet: Background concurrent copying GC freed 16097(1309KB) AllocSpace objects, 13(312KB) LOS objects, 49% free, 2MB/5MB, paused 540us total 287.448ms
2019-03-27 22:33:49.930 2231-2231/fr.free.nrw.commons.beta D/AndroidRuntime: Shutting down VM
2019-03-27 22:33:49.931 2231-2231/fr.free.nrw.commons.beta E/AndroidRuntime: FATAL EXCEPTION: main
    Process: fr.free.nrw.commons.beta, PID: 2231
    java.lang.RuntimeException: Unable to start activity ComponentInfo{fr.free.nrw.commons.beta/fr.free.nrw.commons.auth.LoginActivity}: android.view.InflateException: Binary XML file line #124: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2914)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3049)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6680)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: android.view.InflateException: Binary XML file line #124: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
     Caused by: android.view.InflateException: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
2019-03-27 22:33:49.932 2231-2231/fr.free.nrw.commons.beta E/AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.design.widget.TextInputEditText" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/base.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_dependencies_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_resources_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_0_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_1_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_2_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_3_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_4_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_5_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_6_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_7_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_8_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/base.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_dependencies_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_resources_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_0_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_1_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_2_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_3_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_4_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_5_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_6_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_7_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_8_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_9_apk.apk!/lib/x86, /system/lib, /system/vendor/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.view.LayoutInflater.createView(LayoutInflater.java:606)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:790)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
2019-03-27 22:33:49.937 2231-2231/fr.free.nrw.commons.beta E/AndroidRuntime:     at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
        at com.android.internal.policy.PhoneWindow.setContentView(PhoneWindow.java:420)
        at android.app.Activity.setContentView(Activity.java:2771)
        at fr.free.nrw.commons.auth.LoginActivity.onCreate(LoginActivity.java:109)
        at android.app.Activity.performCreate(Activity.java:7136)
        at android.app.Activity.performCreate(Activity.java:7127)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2894)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3049)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6680)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
    	Suppressed: java.io.IOException: No original dex files found for dex location /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_resources_apk.apk
        at dalvik.system.DexFile.openDexFileNative(Native Method)
        at dalvik.system.DexFile.openDexFile(DexFile.java:354)
        at dalvik.system.DexFile.<init>(DexFile.java:101)
        at dalvik.system.DexFile.<init>(DexFile.java:75)
        at dalvik.system.DexPathList.loadDexFile(DexPathList.java:394)
        at dalvik.system.DexPathList.makeDexElements(DexPathList.java:354)
        at dalvik.system.DexPathList.<init>(DexPathList.java:164)
        at dalvik.system.BaseDexClassLoader.<init>(BaseDexClassLoader.java:74)
        at dalvik.system.BaseDexClassLoader.<init>(BaseDexClassLoader.java:65)
        at dalvik.system.PathClassLoader.<init>(PathClassLoader.java:64)
        at com.android.internal.os.ClassLoaderFactory.createClassLoader(ClassLoaderFactory.java:73)
        at com.android.internal.os.ClassLoaderFactory.createClassLoader(ClassLoaderFactory.java:88)
        at android.app.ApplicationLoaders.getClassLoader(ApplicationLoaders.java:74)
        at android.app.ApplicationLoaders.getClassLoader(ApplicationLoaders.java:40)
        at android.app.LoadedApk.createOrUpdateClassLoaderLocked(LoadedApk.java:727)
        at android.app.LoadedApk.getClassLoader(LoadedApk.java:810)
        at android.app.LoadedApk.getResources(LoadedApk.java:1032)
        at android.app.ContextImpl.createAppContext(ContextImpl.java:2345)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5760)
        at android.app.ActivityThread.access$1100(ActivityThread.java:200)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1651)
        		... 6 more
2019-03-27 22:33:49.941 2231-2231/fr.free.nrw.commons.beta E/ACRA: ACRA caught a RuntimeException for fr.free.nrw.commons.beta
    java.lang.RuntimeException: Unable to start activity ComponentInfo{fr.free.nrw.commons.beta/fr.free.nrw.commons.auth.LoginActivity}: android.view.InflateException: Binary XML file line #124: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2914)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3049)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6680)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: android.view.InflateException: Binary XML file line #124: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
     Caused by: android.view.InflateException: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
2019-03-27 22:33:49.945 2231-2231/fr.free.nrw.commons.beta E/ACRA: Caused by: java.lang.ClassNotFoundException: Didn't find class "android.support.design.widget.TextInputEditText" on path: DexPathList[[zip file "/system/framework/org.apache.http.legacy.boot.jar", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/base.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_dependencies_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_resources_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_0_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_1_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_2_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_3_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_4_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_5_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_6_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_7_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_8_apk.apk", zip file "/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_9_apk.apk"],nativeLibraryDirectories=[/data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/base.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_dependencies_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_resources_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_0_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_1_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_2_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_3_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_4_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_5_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_6_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_7_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_8_apk.apk!/lib/x86, /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_slice_9_apk.apk!/lib/x86, /system/lib, /system/vendor/lib]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        at android.view.LayoutInflater.createView(LayoutInflater.java:606)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:790)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:730)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:863)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:866)
2019-03-27 22:33:49.947 2231-2231/fr.free.nrw.commons.beta E/ACRA:     at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:824)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:515)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:423)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
        at com.android.internal.policy.PhoneWindow.setContentView(PhoneWindow.java:420)
        at android.app.Activity.setContentView(Activity.java:2771)
        at fr.free.nrw.commons.auth.LoginActivity.onCreate(LoginActivity.java:109)
        at android.app.Activity.performCreate(Activity.java:7136)
        at android.app.Activity.performCreate(Activity.java:7127)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1271)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2894)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3049)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6680)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
    	Suppressed: java.io.IOException: No original dex files found for dex location /data/app/fr.free.nrw.commons.beta-XYn_f26K-vcv84IKGspVjA==/split_lib_resources_apk.apk
        at dalvik.system.DexFile.openDexFileNative(Native Method)
        at dalvik.system.DexFile.openDexFile(DexFile.java:354)
        at dalvik.system.DexFile.<init>(DexFile.java:101)
        at dalvik.system.DexFile.<init>(DexFile.java:75)
        at dalvik.system.DexPathList.loadDexFile(DexPathList.java:394)
        at dalvik.system.DexPathList.makeDexElements(DexPathList.java:354)
        at dalvik.system.DexPathList.<init>(DexPathList.java:164)
        at dalvik.system.BaseDexClassLoader.<init>(BaseDexClassLoader.java:74)
        at dalvik.system.BaseDexClassLoader.<init>(BaseDexClassLoader.java:65)
        at dalvik.system.PathClassLoader.<init>(PathClassLoader.java:64)
        at com.android.internal.os.ClassLoaderFactory.createClassLoader(ClassLoaderFactory.java:73)
        at com.android.internal.os.ClassLoaderFactory.createClassLoader(ClassLoaderFactory.java:88)
        at android.app.ApplicationLoaders.getClassLoader(ApplicationLoaders.java:74)
        at android.app.ApplicationLoaders.getClassLoader(ApplicationLoaders.java:40)
        at android.app.LoadedApk.createOrUpdateClassLoaderLocked(LoadedApk.java:727)
        at android.app.LoadedApk.getClassLoader(LoadedApk.java:810)
        at android.app.LoadedApk.getResources(LoadedApk.java:1032)
        at android.app.ContextImpl.createAppContext(ContextImpl.java:2345)
        at android.app.ActivityThread.handleBindApplication(ActivityThread.java:5760)
        at android.app.ActivityThread.access$1100(ActivityThread.java:200)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1651)
        		... 6 more
2019-03-27 22:33:50.079 2231-2231/fr.free.nrw.commons.beta I/Process: Sending signal. PID: 2231 SIG: 9

```

**Device and Android version:** 
Custom Tablet API 28
 
**Commons app version:** 
latest betaDebug

**Would you like to work on the issue?**
Yes

Looking at the log this is probably the key part:

```
    java.lang.RuntimeException: Unable to start activity ComponentInfo{fr.free.nrw.commons.beta/fr.free.nrw.commons.auth.LoginActivity}: android.view.InflateException: Binary XML file line #124: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2914)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3049)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6680)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
     Caused by: android.view.InflateException: Binary XML file line #124: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
     Caused by: android.view.InflateException: Binary XML file line #124: Error inflating class android.support.design.widget.TextInputEditText
```

--- a/app/src/main/res/layout-xlarge/activity_login.xml
+++ b/app/src/main/res/layout-xlarge/activity_login.xml
@@ -121,7 +121,7 @@
                         android:layout_marginStart="@dimen/standard_gap"
                         app:passwordToggleEnabled="true">
 
-                        <android.support.design.widget.TextInputEditText
+                        <com.google.android.material.textfield.TextInputEditText
                             android:id="@+id/login_password"
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"
@@ -145,7 +145,7 @@
                         app:passwordToggleEnabled="false"
                         tools:visibility="visible">
 
-                        <android.support.design.widget.TextInputEditText
+                        <com.google.android.material.textfield.TextInputEditText
                             android:id="@+id/login_two_factor"
                             android:layout_width="match_parent"
                             android:layout_height="wrap_content"

